Vagrantfile

```diff
@@ -106,6 +106,7 @@ Vagrant.configure("2") do |config|
       # Only execute once the Ansible provisioner,
       # when all the machines are up and ready.
       if i == $num_instances
+        config.vm.provision :shell, inline: '[[ -x "$(command -v yum)" ]] && ifup eth1', privileged: true
```

Comment:
The Vagrantfile currently specifies the image should have an [additional private network](https://github.com/kubernetes-incubator/kargo/blob/master/Vagrantfile#L104).  However, Vagrant is buggy, and doesn't bring up interfaces correctly on CentOS/RHEL.

This patch simply ensures, the currently configured Vagrantfile brings up the interface properly on the problematic OS.